app/uk/gov/hmrc/eoricommoncomponent/frontend/models/viewModels/EnrolmentExistsUserStandaloneViewModel.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eoricommoncomponent.frontend.models.viewModels
+
+import play.api.i18n.Messages
+
+object EnrolmentExistsUserStandaloneViewModel {
+
+  def titleAndHeaderLabel(isAdminUser: Boolean)(implicit messages: Messages): String =
+    if (isAdminUser)
+      messages("ecc.eori-exists-user.heading")
+    else
+      messages("ecc.eori-exists-standard-user.heading")
```

Comment:
If we make this a case class rather than an object we can provide the isAdminUser as part of the model?

Reply:
@RobertLaverick I've changed it to a case class, let me know what you think. Thanks!